lookout/style/format/tests/test_quality_report.py

```diff
@@ -148,10 +148,10 @@ def test_eval_aggregate(self):
         self.assertEqual(qcount, 1)
         output = output[:output.find("# Model report for")]
         metrics = _get_metrics(output)
-        expected_metrics = (0.9756371814092953, 0.9756371814092953,
-                            0.8856753997958490, 0.9756371814092953,
-                            0.9284822543249510, 0.9077917659067710,
-                            2668, 2939)
+        expected_metrics = (0.9292385057471264, 0.9292385057471264,
```

Comment:
I see a huge performance drop and it scares me

Reply:
well... 
it is hard to say how our performance affected by test repo with 3000 tokens to predict. 
If you want I can run a quality report for this PR.

Reply:
Yes, please do! I want to understand whether we have a bug or a feature here

Reply:
Besides, do we check the code generation quality in this test? This PR should not change the numbers...

Reply:
did you see the update in the head of PR? https://github.com/src-d/style-analyzer/pull/586#issue-250527761 it changes the numbers because it changes the list of vnodes for the files.

Reply:
Aha, I see. So yeah, we need to carefully investigate. Running the quality reports should be enough for the beginning.

Reply:
So I run it and as a first impression, it looks like we have some unnormal quality drop...
Investigating. will post results when ready.

Reply:
https://github.com/src-d/style-analyzer/pull/586#issuecomment-461347759